bindings/python/pymongoarrow/__init__.py

```diff
@@ -17,10 +17,18 @@
 
 # We must import pyarrow before attempting to load the Cython module.
 import pyarrow as pa  # noqa: F401
-from packaging.version import parse as _parse_version
 
 from pymongoarrow.version import _MIN_LIBBSON_VERSION, __version__  # noqa: F401
 
+try:
+    from packaging.version import parse as _parse_version
+except ImportError:
+    from distutils.version import LooseVersion as _LooseVersion
+
+    def _parse_version(version):
+        return _LooseVersion(version)
+
+
```

Comment:
Accidental addition? Assumed due to this being a scoped docs PR

Reply:
I was getting a warning so I updated it. Do we need a separate ticket?

Reply:
I'd put this as a separate ticket _just_ because unlike with the `__repr__` there's less argument for it being scoped to the docs work.

Reply:
Does that mean that it had to be a separate PR?

Reply:
Done. Resolved?